packages/story-editor/src/components/library/panes/media/common/mediaElement.js

```diff
@@ -181,24 +181,13 @@ function Element({
 
   const onLoad = useCallback(() => setLoaded(true), []);
 
-  const handleKeyDown = useCallback(
-    ({ key }) => {
-      if (key === 'Enter') {
-        onInsert(resource, width, height);
-      } else if (key === ' ') {
-        setIsMenuOpen(true);
-      }
-    },
-    [onInsert, setIsMenuOpen, resource, width, height]
-  );
-
   useKeyDownEffect(
     ref,
     {
       key: ['enter', 'space'],
     },
-    handleKeyDown,
-    [handleKeyDown]
+    () => setIsMenuOpen(true),
+    [setIsMenuOpen]
```

Comment:
Because it's a state setter and we know it to be stable, we can effectively remove it, but I guess it's better to be safe here if someone refactors it to not be stable.